tf-dataflow/README.md

```diff
@@ -0,0 +1,271 @@
+   ![Example](copy-gcs-path.png)
+   Then run...
+   ```bash
+   # Let's make sure we have read access to the GCS objects we need
+   $ gsutil ls "gs://84c6f824-c4ee-4ef2-b654-1030f519e0e9/projects/rut-test/run_executions/urn:run_definitions:El64GVP/urn:run_executions:5M6wq80" | head -5
+   gs://84c6f824-c4ee-4ef2-b654-1030f519e0e9/projects/rut-test/run_executions/urn:run_definitions:El64GVP/urn:run_executions:5M6wq80/urn:app_params:20VD689/
+   # ^ that was OK. Now we'll copy that data over
+   gsutil -m cp -r <paste your path from step 2 here> .
+   cd ..
+   ```
+3. Go back up to where the `dataflow-run.sh` script is stored and run the following. Note that the path (`-p` arg) must be to a directory that contains subdirectories for each `app_param`. In our case, there will only be one.
+   ```bash
+   ./dataflow-run.sh -e local -p urn:run_executions:<your-execution-id> -g <your google project id>
+   ```
+
+   This will produce training and evaluation data in tfrecord format inside of the directory where we copied the app_param. If that looks OK, we can proceed to the next step. If you want to visualize the data, you can use the [tfrecord-viewer](https://github.com/sulc/tfrecord-viewer) project.
+   ```bash
+   $ cd "urn:run_executions:<your-execution-id>"
+   # Example
+   $ find output -type f
+     output/train/72d5852978c4e4ff4c7108946062ae5da8bf4548.tfrecord
+     output/eval/72d5852978c4e4ff4c7108946062ae5da8bf4548.tfrecord
+   ```
+   Note that the `dataflow-run.sh` script is a convenience with some default parameters. You can also run the python script directly. Here's an example:
+   ```bash
+   python main.py --setup_file=./setup.py \
+         --source ${path} \
+         --eval-pct 20 \
+         --runner DataflowRunner \
+         --project ${project} \
+         --region ${region} \
+         --temp_location ${path}/tmp \
+         --staging_location  ${path}/stg \
+         --num_workers ${NUM_WORKERS} \
+         --max_num_workers ${MAX_NUM_WORKERS} \
+         --autoscaling_algorithm THROUGHPUT_BASED
+   ```
+
+#### Real run
+We'll use the same project that we had in step 1 of our local run. With that, let's run:
+```bash
+./dataflow-run.sh -e real -p "gs://projects/<unity project id - GUID>/run_executions/urn:run_definitions:<alnum string>/urn:run_executions:<alnum string>" -g <your google project id>
+```
+
+This will bundle the DataFlow job and run it on GCP. This job assumes that it can parallelize on app-params to increase throughput. **Performance will be poor if all of your data is in a single app param**. Once it's complete, you should be able to move onto the next step of training the actual model. When you run the above command, a bunch of things will be printed to console and it'll continue tailing on your terminal. However, you can cancel out of this at any time and it'll continue running on GCP.
+
+#### (Optional) Convert real world data to Tensorflow format
```

Comment:
For the real world data, we should add link to https://github.com/Unity-Technologies/SynthDet/blob/master/docs/UnityGroceriesReal.md and use `UnityGroceries-Real` as the official name for this dataset.

Reply:
fixed